fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1841,7 +1841,7 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
         long transactionId = stmt.getTxnId();
         TransactionState txnState = null;
         String label = DebugUtil.printId(context.getExecutionId());
```

Comment:
the label is not same as the one used in begin transaction for external olap table.
get label from stmt?

Reply:
done